cache/lru.go

```diff
@@ -88,22 +99,24 @@ func (c *Cache) Get(key Key) (value Value, ok bool) {
 }
 
 // Remove removes the provided key from the cache.
-func (c *Cache) Remove(key Key) {
+func (c *LRUCache) Remove(key Key) {
 	c.mu.Lock()
 	defer c.mu.Unlock()
 	if ele, hit := c.cache[key]; hit {
 		c.removeElement(ele)
 	}
 }
 
-func (c *Cache) removeOldest() {
+func (c *LRUCache) removeOldest() {
```

Comment:
I think it's better to keep coherent functions between LRUCache and RedisCache so either remove this function or implement it in RedisCache

Reply:
In go, the methods that start with an upper-case letter are exported (they can be used from other modules), and methods that start with a lower-case letter are private. Here, `removeOldest` and `removeElement` are just two helpers. The two cache implementations (Redis and LRU) expose the same interface.

Reply:
Oh, ok, thanks.